nav2_pure_pursuit_controller/src/pure_pursuit_controller.cpp

```diff
@@ -20,6 +20,28 @@ using nav2_util::declare_parameter_if_not_declared;
 namespace nav2_pure_pursuit_controller
 {
 
+/**
+ * Find element in iterator with the minimum calculated value
+ */
+template <typename Iter, typename Getter>
+Iter min_by(Iter begin, Iter end, Getter getCompareVal)
+{
+  if (begin == end)
```

Comment:
Use brackets and lint this code

Reply:
Which codestyle are we using? [This one](https://github.com/davetcoleman/roscpp_code_format/blob/master/.clang-format) gives me a very big diff, I think this repo hasn't been formatted yet

Reply:
ament_cpplint and ament_uncrustify

Reply:
I've uncrustified & fix most ament_cpplint issues. There is one error left, that the copyright messages have the wrong format. I'm not sure what to do about those.